src/lib/transaction_logic/mina_transaction_logic.ml

```diff
@@ -2124,11 +2146,11 @@ module Make (L : Ledger_intf.S) : S with type ledger := L.t = struct
       ; burned_tokens
       }
 
-  let apply_transaction ~constraint_constants
+  let apply_transaction ~constraint_constants ~global_slot
       ~(txn_state_view : Zkapp_precondition.Protocol_state.View.t) ledger
       (t : Transaction.t) =
     let previous_hash = merkle_root ledger in
-    let txn_global_slot = txn_state_view.global_slot_since_genesis in
+    let txn_global_slot = global_slot in
```

Comment:
Did you mean here? @mrmr1993

Reply:
No, I meant to pass it into `apply_user_command_unchecked`.